src/components/accession2/properties/CollectedReceivedDate2.tsx

```diff
@@ -4,20 +4,22 @@ import { DatePicker } from '@terraware/web-components';
 import { Accession2, AccessionPostRequestBody } from 'src/api/accessions2/accession';
 import { isInTheFuture } from '@terraware/web-components/utils';
 import strings from 'src/strings';
+import { changeTimezone } from 'src/utils/useTimeZoneUtils';
 
 interface Props {
   onChange: (id: string, value: string) => void;
   record: Accession2 | AccessionPostRequestBody;
   type: 'collected' | 'received';
   validate?: boolean;
+  timeZone?: string;
```

Comment:
this will always be valid since the utils function defaults to UTC

Reply:
I just added this because I didn't know if this component was used in other place. WIll remove

Reply:
i think it is used in the edit view as well

Reply:
oops, don't need to remove.. make it non-optional.

Reply:
Yes, I meant remove the optional, but I have to modify the edit view to do that